sale_order_incoterm_place/report/sale_order_report_templates.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<odoo>
+    <template
+        id="report_saleorder_document_inherit_sale_stock"
+        inherit_id="sale_stock.report_saleorder_document_inherit_sale_stock"
+    >
+        <xpath expr="//p[@t-field='doc.incoterm.code']" position="after">
+            <strong>Place of Destination:</strong>
+            <p t-field="doc.destination_place" />
```

Comment:
```suggestion
        <xpath expr="//p[@t-field='doc.incoterm.code']" position="replace">
            <p>
                <span t-field="doc.incoterm.code"/>
                <span t-field="doc.incoterm_place"/>
            </p>
```
No need to have a dedicated column for the place, which should always be the part of incoterms.